docs/.vitepress/theme/assets/vitepress/theme.css

```
@@ -405,7 +405,8 @@
   --vp-button-brand-active-border: transparent;
   --vp-button-brand-active-text: var(--vp-c-white);
   --vp-button-brand-active-bg: var(--vp-c-brand-1);
-
+  --vp-button-hover-shadow-color: oklch(from var(--vp-c-brand-3) -0.4 c h);
+  /*calc(0.4 * 100% / 100) 1 1); */
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)